### ISSUE TITLE
Possibility to import with an other name

### ISSUE THREAD

**Issue by cedricbonhomme** [mrevutskyi/flask-restless-ng#10]
I managed to solve the issues I had recently (#9). Thank you.

I realize it will be more work than planed to replace my APIv1 based on Flask-Restless in a way that the clients won't notice the changes. So I ended up creating a APIv1.1:
https://github.com/CASES-LU/MOSP/tree/flask-restless-ng/mosp/web/views/api
(the API v2 is based on flask_restx...).

So, my issue now is that [these two dependencies](https://github.com/CASES-LU/MOSP/blob/flask-restless-ng/pyproject.toml#L20-L21) (Flask-Restless and Flask-Restless-ng) must be imported with the same name.

Ofc, I can not do stupid things like:

```python
import flask_restless as flask_restless_old
import flask_restless_ng as flask_restless
```

Would you have an idea to solve this issue ? Maybe I can just put the source code of the un-maintained Flask-Resless in my code or keep my own fork. But I would prefer another solution.

**Comment by mrevutskyi** [comment]
`flask_restless_ng` is intended to be a replacement for `flask_restless`, and not intended to be used simultaneously. In fact, they might conflict with each other.

Why do you need to import the old one?

**Comment by cedricbonhomme** [comment]
I confirm, they conflict ;-)

I am not sure I will be able to provide the exact same API to the clients with Flask-Restless-ng. By clients, I mean real users and also an Angular application. Or it will be a lot of work to achieve this.
This is why for the moment I created the API v1.1, which might replace the API v1. I hope soonish.

I think I will end up forking flask-restless and just call it Flask-RestMoreOrLess with some fixes for recent Flask-SQLalchemy version.

**Comment by mrevutskyi** [comment]
Version 1.0.* of Flask-Restless-NG should be fully API compatible with Flask-Restless version 1.0.0b1

Which version of Flask-Restless are you trying to replace?

**Comment by cedricbonhomme** [comment]
The version 0.17.0, just before 1.0.0b1 :-(

This is what I had to change in the Jinja template of the Flask application (just as example):
https://github.com/CASES-LU/MOSP/commit/6bc38d826c29e4c6bf6da295776613955a543529#diff-c2f855a51fb43b7c60868892f07b6786d1071eacf7b7ad66d193e1be0f4e70bcL79-L92

For example it seems 'objects' is now 'data', in the answer. Also I need to use these "attributes" in order to access to the attributes of the object. I would like to remove this additional level in the result. Same for the "relationships" attribute in the result.

Actually, it works well for me with Flask-Restless-ng. But the structure of the result is quite different. The clients will have to change their code.
Also, what changed is the information in the "meta" (or "metadata") about the pagination, number of results, etc.


I guess, the serialization could help  mehere. If I want the exact same structure (and remove the "attribute" and "relationships" keys).

Actually I am using serialization with Flask-restx in the same project (API v2). Maybe I should try to share the serialization between the APIs.


An an other example (other project), this query:

```bash
https://open-source-security-software.net/api/v1/project?results_per_page=50&q={"filters":[{"name":"licenses","op":"any","val":{"name":"license_id","op":"eq","val":"Apache-2.0"}}]}
```
(it is generated when someone clicks on a Chart.js chart [here](https://open-source-security-software.net/stats/)...) This is pure Falsk-restless filters, without any customization.
These are things I did not have tested yet. But fine, I will see.

I close this issue because it is out of topic. I have the answer about the subject.

I will continue to work in order to fully replace Flask-restless.
